### PR TITLE
Update MenuItemsManager0002Test.php

### PR DESCRIPTION
**Pull request by kshitijSharma2014**

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191  please review it. I hope i meet your expectations.
Tests on options which javier gave me I will add them using a new branch.

File: tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php [hunk 1]
```diff
@@ -8,372 +8,330 @@
...
+		$this->assertFalse($this->menuItemsManagerPage->getRowNumber($title), 'Test menu should not be present');
+		$this->menuItemsManagerPage->addMenuItem($title, $menuType, array('category' => $categoryName, 'Meta Description' => $metaDescription));
+		$message = $this->menuItemsManagerPage->getAlertMessage();
+		$this->assertContains('Menu item successfully saved', $message, 'Menu save should return success', true);
+
+		$homePageUrl = 'index.php';
+		$d = $this->driver;
+		$d->get($cfg->host . $cfg->path . $homePageUrl);
+		$this->siteHomePage = $this->getPageObject('SiteContentFeaturedPage');
+		$this->assetTrue($this->siteHomePage->itemExist($title, 'a'));
```
Comment (puneet0191) [inline tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php:208]: Good!

File: tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php [hunk 2]
```diff
@@ -8,372 +8,330 @@
...
+		$this->assertFalse($this->articleManagerPage->getRowNumber($articleName4), 'Test article should not be present');
+
+		$this->driver->get($cfg->host . $cfg->path . $categoryManager);
+		$this->categoryManagerPage->trashAndDelete($categoryName1);
+		$this->assertFalse($this->categoryManagerPage->getRowNumber($categoryName1), 'Test Category should not be present');
+		$this->categoryManagerPage->trashAndDelete($categoryName2);
+		$this->assertFalse($this->categoryManagerPage->getRowNumber($categoryName2), 'Test Category should not be present');
+
+		$this->driver->get($cfg->host . $cfg->path . $MenuItemsManager);
+		$this->menuItemsManagerPage->setFilter('Menu', 'Main Menu');
```
Comment (puneet0191) [inline tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php:345]: Where is getPageObject?

File: tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php [hunk 3]
```diff
@@ -8,372 +8,330 @@
...
+		$this->assertFalse($this->articleManagerPage->getRowNumber($articleName1), 'Test article should not be present');
+		$this->articleManagerPage->trashAndDelete($articleName2);
+		$this->assertFalse($this->articleManagerPage->getRowNumber($articleName2), 'Test article should not be present');
+		$this->articleManagerPage->trashAndDelete($articleName3);
+		$this->assertFalse($this->articleManagerPage->getRowNumber($articleName3), 'Test article should not be present');
+		$this->articleManagerPage->trashAndDelete($articleName4);
+		$this->assertFalse($this->articleManagerPage->getRowNumber($articleName4), 'Test article should not be present');
+
+		$this->driver->get($cfg->host . $cfg->path . $categoryManager);
+		$this->categoryManagerPage->trashAndDelete($categoryName1);
```
Comment (puneet0191) [inline tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php:338]: getPageObject?

File: tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php [hunk 4]
```diff
@@ -8,372 +8,330 @@
...
+		$this->siteHomePage = $this->getPageObject('SiteContentFeaturedPage');
+
+		$this->siteHomePage->itemClick($title);
+		$this->assertTrue($this->siteHomePage->itemExist($articleName3, 'a'));
+		$this->assertTrue($this->siteHomePage->itemExist($articleName4, 'a'));
+
+
+		$this->doAdminLogin();
+		$this->driver->get($cfg->host . $cfg->path . $articleManager);
+		$this->articleManagerPage->trashAndDelete($articleName1);
```
Comment (puneet0191) [inline tests/system/webdriver/tests/menus/MenuItemsManager0002Test.php:327]: getPageObject?

**Comment by kshitijSharma2014** [comment]
@puneet0191  I have made the changes.

**Pull request merged**